native/swift/Example/Example/WordPressAPI+Extensions.swift

```diff
@@ -11,13 +11,7 @@ extension WordPressAPI {
                 throw CocoaError(.validationMissingMandatoryProperty)
             }
 
-            let parsedUrl = try ParsedUrl.parse(input: defaultSiteUrl)
-            let apiRootUrl = try ParsedUrl.parse(
-                input: parsedUrl
-                    .asURL()
-                    .appendingPathComponent("wp-json")
-                    .absoluteString
-            )
+            let apiRootUrl = try ParsedUrl.parse(input: defaultSiteUrl)
```

Comment:
Maybe I am misunderstanding what is happening here, but this doesn't look correct to me at a first glance. Maybe the name `defaultSiteUrl` is what's confusing me since it's being cast to `apiRootUrl`?

Reply:
Addressed in https://github.com/Automattic/wordpress-rs/pull/658/commits/7aa025b413456829119d0a495b027605f8ae8c7d

Reply:
Thank you! 🙇‍♂️ 

I'll leave the actual review to @jkmassel since this PR is only related to Swift.